docs/chain/collator.md

```diff
@@ -1,136 +1,116 @@
-When this happens your collator will crash.
-Since the WASM is the single truth all collators should execute the WASM version of the state transition function.
+A KILT collator should use the `--execution=wasm` parameter for both the relaychain and parachain collation.
+The alternative to WASM runtime execution is native runtime execution, which might be faster but can, in some cases, deviate from the WASM execution logic and result in a different state.
+When this happens the collator will crash and will stop producing blocks.
+Since the WASM runtime logic is part of the blockchain state itself and hence represents the single source of truth, all collators should execute the WASM version of the runtime logic.
 
-#### Select which runtime and chain to use
+#### Peregrine and Spiritnet chain specs
 
-The `--chain` parameter decides which blockchain we want to execute.
+The `--chain` parameter decides which blockchain the KILT collator will join.
 We need to pass this parameter to the parachain and the relaychain, since both chains are separate blockchains.
-The parachain accepts an additional runtime parameter that selects which runtime is executed and decides which execution environment is used.
+The KILT parachain accepts an additional runtime parameter to select which runtime is executed.
 This can either be `peregrine` or `spiritnet`.
-For now this parameter isn't required, since both runtimes are very similar and can run in the same environment, but it is safer to provided it already now.
+For now this parameter is not required, since both runtimes are very similar and can run in the same environment, but it is safer to provide it already now.
 
 #### Where are all the files stored?
 
 The `--base-path` parameter configures where all the persistent files are written.
-By default the session keys will also be stored int he *base path*, but we recommend to separate them from the other files.
-This makes sure that the keyfiles are not accidentally lost or published when you backup or restore your blockchain database.
-You can configure where to store the session keys using `--keystore-path`.
-Since you only use session keys for the parachain, there is no need to add this to the relaychain part of the command.
+By default the session keys will also be stored in the *base path*, but we recommend to separate them from the other files.
+This makes sure that the keyfiles are not accidentally lost or published when the blockchain database is backed up or restored.
+You can configure where to store the session keys using the `--keystore-path` option.
+Since the collator will collate only for the parachain, there is no need to add this to the relaychain part of the command.
 
-### Acquire the executable
+### Run a Peregrine collator
 
 <Tabs
   groupId="exec-strategy"
   defaultValue="Docker"
   values={[
-    {label: 'Native', value: 'Native'},
+    {label: 'Binary', value: 'Binary'},
```

Comment:
The Docker option doesn't use a binary? I prefer native since that's the difference in my opinion.

Reply:
For me it was confusing as we are not running the native runtime, bus still running the wasm version. Or maybe I got it wrong and Native indicates that the native runtime version is used? Maybe binary is not the best word choice, but I think neither is native if we then use `--execution=wasm`.

Reply:
Ok yes didn't see that it's overloaded with the native runtime vs wasm runtime. good point. Leave it like it is...